### ISSUE TITLE
Extra steps needed to build on Mac OS

### ISSUE THREAD

**Issue by rchoudhary** [NewLunarFire/png2snes#14]
To get this program to build on Mac OS 10.13 (High Sierra) I needed to do the following steps:

  1. Install `argp-standalone` using Homebrew like so:
    `brew install argp-standalone`
  2. Add ` -I/usr/local/include/` to `CFLAGS` and `-largp` to `LDFLAGS` in the Makefile
  3. Add `#include <limits.h>` to the top of `argparser.c`

I believe that Step 1 should be mentioned in the README, and that Steps 2 and 3 should be changed in the Makefile and `argparser.c` respectively.

**Comment by NewLunarFire** [comment]
I agree. If you could submit a PR I will review it. Those steps aren't necessary on Linux, so I will make sure this still builds on Linux before merging.
